src/frameworks/fastify.js

```diff
@@ -10,6 +16,10 @@ function getValidator(validateRequest) {
 
     function validate(request, reply) {
         const requestOptions = _getParameters(request);
+        if (skiplist.includes(requestOptions.path)) {
```

Comment:
this could cause issues when the given string can show up in different path segments.

Reply:
It's an exact match on endpoint url right now; but I guess regexp is a more flexible solution here